docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -380,20 +380,12 @@ <h3>
 		</p>
 
 		<h3>
-			[method:undefined copyTextureToTexture]( [param:Texture srcTexture], [param:Texture dstTexture], [param:Box2 srcRegion], [param:Vector2 dstPosition], [param:Number level] )
-		</h3>
-		<p>
-			Copies the pixels of a texture in the bounds '[page:Box2 srcRegion]' in the destination texture starting from the given position.
-			The `depthTexture` and `texture` property of render targets are supported as well.<br />
-			When using render target textures as `srcTexture` and `dstTexture`, you must make sure both render targets are intitialized e.g. via [page:.initRenderTarget]().
-		</p>
-
-		<h3>
-			[method:undefined copyTextureToTexture3D]( [param:Texture srcTexture], [param:Texture dstTexture], [param:Box3 srcRegion], [param:Vector3 dstPosition], [param:Number level] )
+			[method:undefined copyTextureToTexture]( [param:Texture srcTexture], [param:Texture dstTexture], [param:Box3 srcRegion], [param:Vector3 dstPosition], [param:Number level] )
```

Comment:
This function can technically take Box2 and Vector2 as arguments, as well. Is there a typical way to signify that here?

Reply:
I think we separated multiple types with a `|` so far.

Reply:
Hm it's a bit verbose looking and not super easy to read but I supposed it can be addressed later -

<img width="912" alt="image" src="https://github.com/user-attachments/assets/0d13cc1c-93d2-458c-8ccc-8499cd5100e9">